test/sol6/kyberNetwork.js

```diff
@@ -2074,53 +2073,6 @@ contract('KyberNetwork', function(accounts) {
         });
     });
 
-    describe("test handle change edge case", async function(){
```

Comment:
why these tests removed?

Reply:
no need to mock to test when required is delete